### ISSUE TITLE
A11y_SQL Azure Data Studio_Add an Azure account_Import flat file wizard_Reflow:On zooming 400% then server the database checkbox is not visible completely.

### ISSUE THREAD

**Issue by SwatiTiwari-TCS** [microsoft/azuredatastudio#16307]
"Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

GitHubTags: #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudio_Win32_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG1.4.10;
Pre-Requisite:
Set display resolution

Environment Details:
Application Name: SQL Azure Data Studio
Version: 1.30.0-insider (user setup)
Windows Version: Win10

Repro Steps:

1. Launch the application-Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Hover over to the 'Azure' header and select the '+' icon.
4. Hover over to the Azure header in the blade that opens up and click on the "+" icon.
5. Tab navigate to server the database checkbox and observe its not visible completely on 400% zoom.

Actual:
On zooming 400% then server the database checkbox is not visible completely.

Expected:
On zooming 400% then server the database checkbox should be visible completely.

User Impact:
Low vision user will miss some information if the check box name is not visible on zooming 400%.
![16307](https://user-images.githubusercontent.com/65954864/126135202-ca8dc816-7fdc-4a1a-afc5-630245257023.png)

**Comment by alanrenmsft** [comment]
closing the issue, Azure Data Studio requested exception for WCAG 2.1: https://ceacc.visualstudio.com/ceaccex/_workitems/edit/42844/
